src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala

```diff
@@ -167,10 +166,11 @@ abstract class SmartDataLakeBuilder extends SmartDataLakeLogger {
     logger.info(s"Application: ${appConfig.applicationName}")
     logger.info(s"Master: ${appConfig.master}")
     logger.info(s"Deploy-Mode: ${appConfig.deployMode}")
+    val appName = appConfig.applicationName.getOrElse(appConfig.feedSel)
```

Comment:
Did you try with different regular expressions if everything still works? i.e when using ".*" as feed selector